packages/eslint-config/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "main": "index.js",
   "devDependencies": {
+    "eslint-plugin-daily": "*",
```

Comment:
Why is this different than this `@dailydotdev/eslint-plugin-daily-dev-eslint-rules`?